tests/RobotFramework/tests/cumulocity/telemetry/child_device_telemetry.robot

```diff
@@ -48,6 +48,13 @@ Child devices support sending custom events overriding the type
     Log    ${events}
 
 
+Child devices support sending large events
+    Execute Command    tedge mqtt pub te/device/${CHILD_SN}///e/largeEvent "$(printf '{"text":"Large event","large_text_field":"%s"}' "$(yes "x" | head -n 100000 | tr -d '\n')")"
+    ${events}=    Device Should Have Event/s    expected_text=Large event    with_attachment=False    minimum=1    maximum=1    type=largeEvent    fragment=large_text_field
+    Length Should Be    ${events[0]["large_text_field"]}    100000
```

Comment:
Wouldn't it be better to assert the MQTT messages also, to assert that that the converted message is not present?

Reply:
Since the robot test should not be affected by c8y's requirement, this test might publish via MQTT. No MQTT message is covered in the Rust test.

Reply:
Yes that was my call. The system test does not care that http was used, it more cares that a large message can be sent to c8y from the device.